osu.Game/Screens/Play/Player.cs

```diff
@@ -157,14 +157,16 @@ protected override void LoadComplete()
                 PrepareReplay();
         }
 
-        private Replay recordingReplay;
+        private Score recordingScore;
 
         /// <summary>
         /// Run any recording / playback setup for replays.
         /// </summary>
         protected virtual void PrepareReplay()
         {
-            DrawableRuleset.SetRecordTarget(recordingReplay = new Replay());
+            DrawableRuleset.SetRecordTarget(recordingScore = new Score { Replay = new Replay() });
```

Comment:
I think this could just as well be

```suggestion
            DrawableRuleset.SetRecordTarget(recordingScore = new Score());
```

as `Score` creates its own anyway (spotted this after being confused why the next line doesn't seem to check for null score info, but this one creates the replay explicitly).